src/lazy/encoder/binary/v1_1/container_writers.rs

```diff
@@ -1,13 +1,14 @@
 use bumpalo::collections::Vec as BumpVec;
 use bumpalo::Bump as BumpAllocator;
 use delegate::delegate;
+use ice_code::ice as cold_path;
```

Comment:
That's good feedback, thanks. I was wondering if I should push the name `cold_path` upstream since it's a bit more self-documenting (albeit less fun 😛). I'll leave this for the moment since it's a pretty easy find/replace to change later. @nirosys, do you have an opinion one way or another on the macro name?

Reply:
I do like the self-documenting nature of `cold_path`, since it's a bit easier to understand without familiarity of what ice_cold does. But, I don't feel too strongly about it, since it does rely on knowing what a cold path is to be more understandable.